### ISSUE TITLE
Delete images from a data sourse

### ISSUE THREAD

**Issue by AlexDibrivnyy** [Labelbox/Labelbox#153]
Good day,

We would like to ask you to add a "Delete images from dataset" feature to a LabelBox. 

Background is:
We added 300 new images to our dataset and, such as, they were 3000*4000 px size, they were rotated on a Labeling panel. We decided to decrease their size and re-uplaod, but did not find a way to remove queued images from a dataset.
Removing dataset will not help us, because we have another 250 annotated images on this dataset.

To summarize:
- Remove images from dataset.
- Why big images are rotated on a Labeling tool?

Thank you for your time. Waiting for your response.

Thank you.
